content/docs/cli/commands/pulumi_convert.md

```diff
@@ -14,6 +14,13 @@ Convert Pulumi programs from a supported source program into other supported lan
 
 The source program to convert will default to the current working directory.
 
+### Supported Languages
```

Comment:
I could be wrong, but I don't think we need to expose PCL here. PCL isn't documented (I believe that's intentional) as it is an internal representation that Pulumi uses.